orbs/docker/orb.yml

```diff
@@ -90,7 +94,7 @@ commands:
               --no-progress \
               -f json \
               -o /tmp/trivy.json \
-              <<parameters.image_name>>:<<parameters.image_tag>>
+              <<parameters.image_name>>:<<parameters.image_tag>> || <<parameters.optional>>
```

Comment:
Not necessarily a blocker but we'll lose the status code if optional is set to false:

```
bash: lkdsajflasdkfj: command not found
bash-3.2$ echo $?
127
bash-3.2$ laksdjflaksdjf || false
bash: laksdjflaksdjf: command not found
bash-3.2$ echo $?
1
```

I wonder if there's an implem that would let us keep it 🤔